index.html

```diff
@@ -289,6 +289,80 @@ <h2>About me</h2>
           </ul>
         </div>
       </div>
+
+      <form class="contact-form">
+        <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+
+        <input type="text" placeholder="Full name" />
+        <input type="text" placeholder="Email address"/>
+        <textarea placeholder="Enter text here"></textarea>
+
+        <button type="submit">Get in touch</button>
+      </form>
```

Comment:
Your form is not sending a POST REQUEST to the server. Please consider using the [Formspree service](https://formspree.io/html) for sending user data to the server to store it so people coming to your portfolio can interact with you efficently.